data/settings/1.19.x/ecs.toml

```diff
@@ -17,6 +17,16 @@ description = "Settings related to logs from the ECS Agent"
 
 
 # reference
+[[docs.ref.backend-host]]
+description = "Sets the endpoint to make calls against (e.g `ecs.us-east-1.amazonaws.com`)."
+default = "The endpoint for your current region."
```

Comment:
You could add a link to the ECS docs for this setting too.

Reply:
Humm. Got a good link to that?

Reply:
Talked about this offline, the ECS folks don't have documentation about it.